operations/src/main/java/com/nextdoor/bender/operations/geo/GeoIpOperation.java

```diff
@@ -73,6 +73,13 @@ public InternalEvent perform(InternalEvent ievent) {
      * Sometimes the field contains comma separated ip addresses (ie forwarded web requests). In
      * this case pick the first value in the list which is typically the user.
      */
+    String ipStr = null;
+    if (ipObj instanceof String) {
```

Comment:
You repeat this a few times.. perhaps it would be easier to create a `getFieldAsString()` method that does this?

Reply:
Agreed